spec/migrations/20210718074605_update_ibm_vendor_to_ibm_cloud_spec.rb

```diff
@@ -0,0 +1,29 @@
+require_migration
+
+RSpec.describe UpdateIbmVendorToIbmCloud do
+  let(:vm_stub) { migration_stub(:Vm) }
+
+  migration_context :up do
+    it "updates vendor to 'ibm_cloud' for IBM Cloud managers" do
+      vpc_vm = vm_stub.create!(:type => "ManageIQ::Providers::IbmCloud::VPC::CloudManager::Vm", :vendor => "ibm")
+      powervs_vm = vm_stub.create!(:type => "ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::Vm", :vendor => "ibm")
```

Comment:
Can you also add a _non_ VPC Vm, and also verify that it _doesn't_ get changed post-migration?

Reply:
I added checks for both OpenStack and PowerVC type to ensure they don't get changed (seemed like the most likely type to have something IBM related and thus be inadvertently changed).